.github/ISSUE_TEMPLATE/release.md

```diff
@@ -0,0 +1,35 @@
+---
+name: Release Checklist
+about: Create release checklist
+---
```

Comment:
Note that because you've added this header, this repo's "New issue" button now points to a [template chooser](https://github.com/openshift/cluster-monitoring-operator/issues/new/choose), which may not be what you wanted.

In CoreOS land, we omit the header, which prevents the template chooser from being enabled.  Then we put a link of the form https://github.com/openshift/cluster-monitoring-operator/issues/new?template=release.md in the developer documentation.

Reply:
Thanks for your reply!

cc @paulfantom @s-urbaniak

Reply:
thanks @bgilbert! i guess we'll fix this here then :-)